.github/workflows/docker-publish.yml

```diff
@@ -56,3 +51,10 @@ jobs:
           platforms: linux/amd64,linux/arm64
           tags: ghcr.io/${{ github.repository }}:${{ github.event.release.tag_name }}, ghcr.io/${{ github.repository }}:latest
           labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Package and Push Chart
+        run: |
+          helm plugin install https://github.com/chartmuseum/helm-push.git
+          helm repo add chartmuseum https://charts.ietf.org
+          helm cm-push --version="${{github.event.release.tag_name}}" --username="${{secrets.HELM_REPO_USERNAME}}" --password="${{secrets.HELM_REPO_PASSWORD}}" helm/ chartmuseum
```

Comment:
You'll have to ensure the tag name is not prefixed with a `v`. I see that the current releases are using a `v` prefix which won't work with helm charts.

Reply:
Fixed with `"${{github.event.release.tag_name | sed 's/^v//'}}"`